field/composite_test.go

```diff
@@ -595,142 +635,6 @@ func TestCompositePackingWithTags(t *testing.T) {
-					Pref:        prefix.ASCII.Fixed,
-				}),
-				"2": NewNumeric(&Spec{
-					Length:      1,
-					Description: "Reserved for Future Use",
-					Enc:         encoding.ASCII,
-					Pref:        prefix.ASCII.Fixed,
-				}),
-				"3": NewString(&Spec{
-					Length:      1,
-					Description: "POS Terminal Location",
-					Enc:         encoding.ASCII,
-					Pref:        prefix.ASCII.Fixed,
-				}),
-				"4": NewString(&Spec{
-					Length:      1,
-					Description: "POS Cardholder Presence",
-					Enc:         encoding.ASCII,
-					Pref:        prefix.ASCII.Fixed,
-				}),
-				"5": NewString(&Spec{
-					Length:      1,
-					Description: "POS Card Presence",
-					Enc:         encoding.ASCII,
-					Pref:        prefix.ASCII.Fixed,
-				}),
-				"6": NewString(&Spec{
-					Length:      1,
-					Description: "POS Card Capture Capabilities",
-					Enc:         encoding.ASCII,
-					Pref:        prefix.ASCII.Fixed,
-				}),
-				"7": NewString(&Spec{
-					Length:      1,
-					Description: "POS Transaction Status",
-					Enc:         encoding.ASCII,
-					Pref:        prefix.ASCII.Fixed,
-				}),
-				"8": NewString(&Spec{
-					Length:      1,
-					Description: "POS Transaction Security",
-					Enc:         encoding.ASCII,
-					Pref:        prefix.ASCII.Fixed,
-				}),
-				"9": NewNumeric(&Spec{
-					Length:      1,
-					Description: "Reserved for Future Use",
-					Enc:         encoding.ASCII,
-					Pref:        prefix.ASCII.Fixed,
-					Pad:         padding.Left('0'),
-				}),
-				"10": NewString(&Spec{
-					Length:      1,
-					Description: "Cardholder-Activated Terminal Level",
-					Enc:         encoding.ASCII,
-					Pref:        prefix.ASCII.Fixed,
-				}),
-				"11": NewString(&Spec{
-					Length:      1,
-					Description: "POS Card Data Terminal Input Capability Indicator",
-					Enc:         encoding.ASCII,
-					Pref:        prefix.ASCII.Fixed,
-				}),
-				"12": NewString(&Spec{
-					Length:      2,
-					Description: "POS Authorization Life Cycle",
-					Enc:         encoding.ASCII,
-					Pref:        prefix.ASCII.Fixed,
-				}),
-				"13": NewString(&Spec{
-					Length:      3,
-					Description: "POS Country Code",
-					Enc:         encoding.ASCII,
-					Pref:        prefix.ASCII.Fixed,
-				}),
-				"14": NewString(&Spec{
-					Length:      10,
-					Description: "POS Postal Code",
-					Enc:         encoding.ASCII,
-					Pref:        prefix.ASCII.Fixed,
-				}),
-			},
-		}
-
-		data := &ExcessSubfieldsTestData{}
-
-		composite := NewComposite(excessSubfieldsSpec)
-		err := composite.SetData(data)
-		require.NoError(t, err)
-
-		// Subfield 12, 13 & 14 fall outside of the bounds of the
-		// 11 byte limit imposed by the prefix. [011 | 10000100012]
-		// Therefore, it won't be included in the packed bytes.
-
-		packed := []byte("01110000100012")
-
-		read, err := composite.Unpack(packed)
-
-		require.NoError(t, err)
-		require.Equal(t, 14, read)
-
-		packedBytes, err := composite.Pack()
```

Comment:
Now we just pack it back. Unset fields will not be included in the packed value. It's just expected behavior.

So, test is removed as we don't test here anything. Or I'm missing something.